packages/pieces/community/google-drive/src/lib/common/get-file-content.ts

```
@@ -52,14 +52,14 @@ const googledlCall = async (
       )
     );
 
-  const extention = '.' + extension(mimeType);
-  const srcFileName = fileName ?? fileId + extention;
+  const extension = '.' + extension(mimeType);
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)